internal/clients/idenfy/idenfy.go

```diff
@@ -0,0 +1,121 @@
+/*
+Package idenfy contains the iDenfy client for the application.
+This layer is responsible for interacting with the iDenfy API. the main operations are:
+- creating a verification session
+- verifying the callback signature
+*/
+package idenfy
+
+import (
+	"context"
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"github.com/threefoldtech/tf-kyc-verifier/internal/models"
+	"github.com/valyala/fasthttp"
```

Comment:
Why? https://github.com/valyala/fasthttp#faq

```
Are there known net/http advantages comparing to fasthttp?

Yes:

net/http supports HTTP/2.0 starting from go1.6.
net/http API is stable, while fasthttp API constantly evolves.
net/http handles more HTTP corner cases.
net/http can stream both request and response bodies
net/http can handle bigger bodies as it doesn't read the whole body into memory
net/http should contain less bugs, since it is used and tested by much wider audience.
```